src/lib/core/media-observer/media-observer.ts

```diff
@@ -43,41 +41,67 @@ import {BreakPointRegistry, OptionalBreakPoint} from '../breakpoints/break-point
  *    status: string = '';
  *
  *    constructor(mediaObserver: MediaObserver) {
- *      const onChange = (change: MediaChange) => {
- *        this.status = change ? `'${change.mqAlias}' = (${change.mediaQuery})` : '';
- *      };
+ *      const media$ = mediaObserver.asObservable().pipe(
+ *        filter((changes: MediaChange[]) => true)   // silly noop filter
+ *      );
  *
- *      // Subscribe directly or access observable to use filter/map operators
- *      // e.g. mediaObserver.media$.subscribe(onChange);
+ *      media$.subscribe((changes: MediaChange[]) => {
+ *        let status = '';
+ *        changes.forEach( change => {
+ *          status += `'${change.mqAlias}' = (${change.mediaQuery}) <br/>` ;
+ *        });
+ *        this.status = status;
+ *     });
  *
- *      mediaObserver.media$()
- *        .pipe(
- *          filter((change: MediaChange) => true)   // silly noop filter
- *        ).subscribe(onChange);
  *    }
  *  }
  */
 @Injectable({providedIn: 'root'})
 export class MediaObserver {
+
   /**
-   * Whether to announce gt-<xxx> breakpoint activations
+   * @deprecated Use `asObservable()` instead.
+   * @deletion-target v7.0.0-beta.24
```

Comment:
deletion-target should be +2 releases which would put this at beta.26 (assuming it's released with beta 24)

Reply:
deprecation for 24
deletion for 25

Reply:
Compromise accepted 😄